packages/matter-node.js/package.json

```diff
@@ -27,7 +27,9 @@
         "clean": "rm -rf dist && rm *.tsbuildinfo || true",
         "build": "tsc --project tsconfig.dist.json",
         "build-clean": "npm run clean && npm run build",
-        "test": "TZ=utc jest"
+        "test": "TZ=utc jest --maxWorkers=5",
```

Comment:
Maybe can be moved to jest.config?

Reply:
yes works, adjusted - also to 50% to be more generic